pkg/meta/tkv.go

```diff
@@ -1910,11 +1910,29 @@ func (m *kvMeta) CopyFileRange(ctx Context, fin Ino, offIn uint64, fout Ino, off
 func (m *dbMeta) cleanupDeletedFiles() {}
 
 func (m *dbMeta) cleanupSlices() {}
-
-
-func (m *dbMeta) deleteChunk(inode Ino, indx uint32) error {}
 */
 
+func (m *kvMeta) deleteChunk(inode Ino, indx uint32) error {
+	key := m.chunkKey(inode, indx)
+	var slices []*slice
+	err := m.txn(func(tx kvTxn) error {
+		buf := tx.get(key)
+		slices = readSliceBuf(buf)
+		tx.set(key, []byte{})
```

Comment:
tx.dels(key)

Reply:
hmm, to be aligned with redis implementation, should we leave the key with empty value there? Though i think there is no big difference.

Reply:
they will be leaked in TiKV. Redis will cleanup the empty keys, but other KV will not.